File: PRAKTIKUM MODUL 6/PRAK603-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main(){
    int g, z, angka, i;
    scanf("%d %d", &g, &z);
    
    if(g!=z){
        printf("Jumlah tidak sama");
    }else{

    int barisx[g];int barisy[z];
    for(i = 0; i < g; i++){
        scanf("%d", &barisx[i]);} 
    for (i = 0; i < z; i++){
        scanf("%d", &barisy[i]);} 
    for(i = 0; i < g; i++){
        angka = barisx[i] * barisy[i];
        printf("%d ", angka);}
    }
}

File: PRAKTIKUM MODUL 6/PRAK603-2410817310009-Muhammad_Ghazi_Rakhmadi.py
g, z = map(int, input().split())
if (g != z):
    print('Jumlah tidak sama')
else:
    barisx = list(map(int, input().split()))
    barisy = list(map(int, input().split()))
    for i in range (g):
        print(barisx[i] * barisy[i], end=' ')